spec/views/groups/index.html.haml_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+RSpec.describe "groups/index", type: :view do
+  before(:each) do
+    assign(:groups, [
+      Group.create!(
+        :category => "Category",
+        :boolean => ""
+      ),
+      Group.create!(
+        :category => "Category",
+        :boolean => ""
+      )
+    ])
+  end
+
+  it "renders a list of groups" do
+    render
+    assert_select "tr>td", :text => "Category".to_s, :count => 2
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.